### DISCUSSION TITLE
How to debug star/favorites rating

### DISCUSSION THREAD

**Discussion by JOJ0** [navidrome/navidrome#2338] [Q&A] [answered]
Starred tracks and albums I add in the iOS app play:sub are not appearing in the Navidrome WebUI. I'm not sure where the problem lies. I think both components should support the syncing/uploading from subsonic client to server.

What would I need to look for in
- the regular Navidrome log
- the Navidrome debug log

Thanks for any hints on how to proceed. Or maybe this isn't supported and I read the docs of both applications wrong?

**Comment by deluan** [comment, marked as answer]
Stars and ratings should be in sync. To make sure play:Sub is sending the requests to Navidrome, change the log_level to debug. When starring/unstarring things in play:Sub, you should see logs like these (here using airsonic-refix):
```
DEBU[0069] API: New request /rest/star                   client="https://airsonic.netlify.app" requestId=Deluans-MacBook-Pro.local/NUBhWmkf7b-000021 username=admin version=1.15.0
DEBU[0069] Changing starred                              ids="[cfb1a8e992a054495eedd60e1b15e5f3]" requestId=Deluans-MacBook-Pro.local/NUBhWmkf7b-000021 starred=true username=admin
DEBU[0069] API: Successful response                      endpoint=/rest/star requestId=Deluans-MacBook-Pro.local/NUBhWmkf7b-000021 status=OK username=admin
DEBU[0083] API: New request /rest/unstar                 client="https://airsonic.netlify.app" requestId=Deluans-MacBook-Pro.local/NUBhWmkf7b-000022 username=admin version=1.15.0
DEBU[0083] Changing starred                              ids="[cfb1a8e992a054495eedd60e1b15e5f3]" requestId=Deluans-MacBook-Pro.local/NUBhWmkf7b-000022 starred=false username=admin
DEBU[0083] API: Successful response                      endpoint=/rest/unstar requestId=Deluans-MacBook-Pro.local/NUBhWmkf7b-000022 status=OK username=admin
```
If you can't see these requests in the logs, this is probably an issue with play:Sub configuration, and you should reach out their support.

**Reply by JOJ0** [reply]
Thanks so much for clarifying. I just now realized that it was user error. I usually star single tracks, not albums. The "favorites" menu entry on Navidrome UI shows starred albums and not starred tracks. 

In the songs view I can see all my starred tracks. Everything is working as expected.

Do you have plans to support a view of starred tracks in the UI?

**Reply by JOJ0** [reply]
Oh, it's user error again. I am looking for track stars in the album menu of the sidebar. That can't work.

I found the filter option while browsing songs and can now see all my stars and can add them to a playlist, which was my goal in the end. 

Nevermind and thanks a lot!
